src/aznfswatchdog

```diff
@@ -133,24 +133,29 @@ check_for_regional_account()
 # we delete the entry so that kernel looks up fresh NAT rules and creates a new entry.
 #
 declare -A cthash_synsent
-declare -A cthash_synsent_now
 
 reconcile_conntrack_synsent()
 {
     local l_ip=$1
     local l_sport=$2
     local l_dport=$3
     local l_nfsip=$4
+    local seconds_remaining=$5
 
     key="${l_ip}:${l_sport}:${l_dport}:${l_nfsip}"
-    let cthash_synsent[$key]++
-    let cthash_synsent_now[$key]++
+    
+    if [ ! -v cthash_synsent[$key] ]; then
+        cthash_synsent[$key]=$seconds_remaining
+        return
+    fi
+
+    age_seconds=$(expr ${cthash_synsent[$key]} - $seconds_remaining)
```

Comment:
`#`
`#` How long has this entry been around?
`#` If it's around for more than 25-30 secs, we consider the entry as "stuck" and delete it to cause fresh entry to
`#` be created, and help make progress.
`#`

you can remove the comment block which is now incorrect and move this comment block above the age_seconds= line.